server.go

```diff
@@ -567,6 +566,25 @@ func serverListener_unix() (net.Listener, error) {
 		return nil, err
 	}
 
+	// By default, unix sockets are only writable by the owner. Set up a custom
+	// group owner and group write permissions if configured.
+	if groupString := os.Getenv(EnvUnixSocketGroup); groupString != "" {
+		group, err := strconv.Atoi(groupString)
```

Comment:
Maybe it might be more intuitive to support group names as well, e.g., using https://pkg.go.dev/os/user@go1.21.0#LookupGroup ?

Reply:
That's a good call - added in 24eea0627f90d63f35d71fdb71206216b86608ca